build.rs

```diff
@@ -475,10 +481,6 @@ fn generate_bindings(r_paths: &InstallationPaths, version_info: &RVersionInfo) {
     let target_os = std::env::var("CARGO_CFG_TARGET_OS").unwrap();
     let target_arch = env::var("CARGO_CFG_TARGET_ARCH").unwrap();
 
-    println!(
-        "Generating bindings for target: {target}, os: {target_os}, architecture: {target_arch}"
-    );
-
```

Comment:
i believe this is very helpful honestly, and some things should definitely be there in case we want people to run it with `-vv` flag to inspect it..